easybuild/easyconfigs/f/fineSTRUCTURE/fineSTRUCTURE-2.1.3-foss-2023a.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'fineSTRUCTURE'
+version = '2.1.3'
+
+homepage = 'https://people.maths.bris.ac.uk/~madjl/finestructure/finestructure_info.html'
+description = """fineSTRUCTURE is a fast and powerful algorithm for identifying population structure using
+ dense sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

doesn't look like it depends on blas/lapack or MPI, so please drop it to `GCC` (filename will need changing to match)
```suggestion
toolchain = {'name': 'GCC', 'version': '12.3.0'}
```

---

Thanks for spotting that and done.